server/gateways/stripe.js

```diff
@@ -20,7 +20,7 @@ const getOrCreatePlan = (stripeAccount, plan) => {
 
   return stripeClient.plans.retrieve(plan.id)
     .catch((err) => {
-      if (err.type === 'StripeInvalidRequest' && _.contains(err.message, 'No such plan')) {
+      if (err.type === 'StripeInvalidRequestError' && _.contains(err.message, 'No such plan')) {
```

Comment:
I'd add a `console.log` to make sure you log potential errors even if they don't match that specific error that you expect.